code-challenges.js

```diff
@@ -33,19 +68,51 @@ var myForNumbers2 = [2, 5, -8, 20, 32]
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+//***psudo***
+//function that takes an array
+//create a variable to hold new array
+//use map to multiply each number by 5
+
+// const timesFiveMap = (array) => {
+//   let mappedFive = array.map(value => {
+//     return value * 5
+//   })
+//   return mappedFive
+// }
 
+// **refactor without variable**
 
+const timesFiveMap = (array) => {
+  return array.map(value => {
```

Comment:
```suggestion
  return array.map(value => value * 5);
```
you can use implicit return when using arrow functions for short return statements